branchLayout/api/src/main/java/com/virtuslab/branchlayout/api/BranchLayout.java

```diff
@@ -29,6 +30,28 @@ private static List<IBranchLayoutEntry> collectEntriesRecursively(IBranchLayoutE
     return entryByName.get(branchName).getOrNull();
   }
 
+  @Override
+  public @Nullable IBranchLayoutEntry findNextEntry(String branchName) {
```

Comment:
Change the base of this PR to `get-next-prev-branch`. Then once you merge the PR #1184, this PR #1167 will automatically retarget to `develop`

Reply:
(there's also `git machete github retarget-pr` in CLI that sets the base of the PR in GitHub to what's in your local .git/machete)